client/my-sites/plans-v2/selector.tsx

```diff
@@ -46,6 +46,10 @@ const SelectorPage = ( {
 	const [ productType, setProductType ] = useState< ProductType >( SECURITY );
 	const [ currentDuration, setDuration ] = useState< Duration >( defaultDuration );
 
+	useMemo( () => {
```

Comment:
This should be replaced with `useEffect` since we want to re-run that code every time `defaultDuration` changes. `useMemo` does a similar thing but it also returns the result so we can store it in a variable. We would normally use `useMemo` like this:
```
const result = useMemo( () => anExpensiveComputation( dep1, dep2 ), [ dep1, dep2 ] );
```

Reply:
Ahh, yes, oops, sounds good! 👍